### ISSUE TITLE
Career testimonials, fix for non-English languages

### ISSUE THREAD

**Issue by bosschaert** [hlxsites/sunstar#80]
The hero-career block and career-carousel have hardcoded English language for a number of labels and image alt texts. This should be picked up from the placeholders so that we can use these blocks for other languages as well.
